plugins/flatpak/gs-flatpak.c

```diff
@@ -3864,6 +3910,13 @@ gs_flatpak_app_install (GsFlatpak *self,
 			return FALSE;
 		}
 
+		/* Check disk space requirements for the list */
+		if (!gs_flatpak_allow_install_or_update (self, app, list, FALSE)) {
+			g_debug ("Skipping installation for: %s as prone to failure due to disk-space",
+				  gs_app_get_id (app));
```

Comment:
Always log using the app's unique id (otherwise we may not know which app it is, if more than one have the same ID).